index.tpl.html

```diff
@@ -31,6 +31,7 @@ <h1>
 					<div property="description" class="markdown">Description</div>
 					<a href="?tag=[tag]" property="tag" mv-attribute="null" mv-multiple></a>
 					<meta property="repo" mv-edit-placeholder="E.g. [author]/mavo-[id]">
+					<meta property="url" mv-edit-placeholder="E.g. for self-hosted plugin">
```

Comment:
That is not an example, so I think the UI could be confusing. How about generating a default Github Pages URL from the user's username and the repo? (as a placeholder, not as a default)
Also, we should probably not show this field when the plugin is not hosted in a separate repo.

Reply:
Excellent idea. Thank you.